### ISSUE TITLE
Let user set parameters for program

### ISSUE THREAD

**Issue by andulv** [64bandil/merlin400#8]
Extract (program 1 and 5):
- Soak time

Decarb (program 2):
 - Time
 - (Temperature?)

**Comment by chandz102** [comment]
yeah that sounds good, then we can choose a longer decarb time at a lower temp if we want!

**Comment by chandz102** [comment]
what is the default temp set to now for program 2?

**Comment by andulv** [comment]
Here are defaults for config values that might be of interest:

"distillation_temperature": "125"
"soak_time_seconds": "10"
  "number_of_flushes": "1",
  "flush_time": "10",

DECARB:
      "temperature": "125",
      "time_minutes": "30",

OIL_MIX:
      "temperature": "60",
      "time_minutes": "10",

**Comment by chandz102** [comment]
Nice, I think have both temp and time for decarb would be great to set manual.  Also soak time Sent from my iPhoneOn 25 May 2024, at 20.10, Anders Ulveseth ***@***.***> wrote:﻿
Here are defaults for config values that might be of interest:
"distillation_temperature": "125"
"soak_time_seconds": "10"
"number_of_flushes": "1",
"flush_time": "10",
DECARB:
"temperature": "125",
"time_minutes": "30",
OIL_MIX:
"temperature": "60",
"time_minutes": "10",

**Comment by chandz102** [comment]
Hi Anders! Just checking in, how are things goings?
